src/components/Tooltip/README.md

```diff
@@ -74,6 +74,18 @@ Use only when necessary to provide an explanation for an interface element.
 </div>
 ```
 
+### Light tooltip
+
+Changes the background color to white.
```

Comment:
Could this include a usage guideline, rather than being a description of what this example does?

Reply:
For sure! I chatted with @sarahill and @sadiesaurus and we came up with,

> Light tooltip draws less attention than the default tooltip. Use only when necessary to provide an explanation for an interface element with less emphasis.